rakelib/yardoc.rake

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 desc 'run yardoc against changed files'
-task yardoc: :environment do
+task :yardoc do # rubocop:disable Rails/RakeEnvironment
```

Comment:
This prevents the task from loading up the whole rails app when it runs (there are other things happening the Rakefile that are undesirable but for now this removes reliance on pdftk-java)